tensorpack/tfutils/scope_utils.py

```diff
@@ -12,11 +12,15 @@
 __all__ = ['auto_reuse_variable_scope', 'cached_name_scope', 'under_name_scope']
 
 
-def auto_reuse_variable_scope(func):
+def auto_reuse_variable_scope(func, per_variable=False):
```

Comment:
That's not the syntax of a decorator

Reply:
Sorry. Then it just a one-liner pull-request

Reply:
This seems to fix the travis-ci issue.